ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -193,6 +193,8 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
     public incrementStep(scroll: boolean = true): void {
         const nextIndex = this.nextAvailableSectionIndex();
         if (nextIndex !== -1) {
+            //validationRequested needs to be reset before starting validation.
+            this.validationRequested=false;
```

Comment:
Good catch! 👍

Reply:
Actually, it is the bug fix. 

Other stuff is just an improvement/refactoring.